lib/gds-components/autocomplete/Autocomplete.tsx

```diff
@@ -0,0 +1,39 @@
+import AccessibleAutocomplete, {
```

Comment:
autocomplete is not coming from GDS actually so maybe it shouldn't live in this folder

Reply:
I agree